components/StyledInputFormikField.js

```diff
@@ -98,6 +111,9 @@ StyledInputFormikField.propTypes = {
   display: PropTypes.any,
   width: PropTypes.any,
   flexGrow: PropTypes.any,
+  showError: PropTypes.bool,
+  min: PropTypes.any,
+  max: PropTypes.any,
```

Comment:
number ?

Reply:
A string is also valid. This is lazy me now willing to invest energy in proptypes knowing that we're moving the code to typescript. Will update to `PropTypes.oneOfType([PropTypes.string, PropTypes.number])`.